packages/elastic/src/types.ts

```diff
@@ -7,15 +7,20 @@ import type {
 } from '@debugr/core';
 import type { ClientOptions } from '@elastic/elasticsearch';
 
+export type EntryTransformer<
+  TTaskContext extends TContextBase = TContextBase,
+  TGlobalContext extends TContextShape = TContextShape,
+> = (
+  entry: ReadonlyRecursive<LogEntry<TTaskContext, TGlobalContext>>,
+) => Record<string, any> | undefined | null | false | 0 | '';
```

Comment:
lol, ty vyjmenovany falsy values :D. Nech tam jen undefined :). Nema smysl se v typovani takto rozepisovat, to akorat navadi k prasarnam v kodu